pyfmi.fmi.FMUException: Failed to retrieve the continuous states

On [a FMI2.0 model generated by OpenModelica (SinusGenerator.fmu.zip)](https://github.com/modelon-community/PyFMI/files/6025333/SinusGenerator.fmu.zip), I have the following exception with version 2.8.5:
```
Traceback (most recent call last):
  File "./fmi-comgw-connector.py", line 12, in <module>
    res = model.simulate() #final_time=10)
  File "src/pyfmi/fmi.pyx", line 7873, in pyfmi.fmi.FMUModelME2.simulate
  File "src/pyfmi/fmi.pyx", line 319, in pyfmi.fmi.ModelBase._exec_simulate_algorithm
  File "src/pyfmi/fmi.pyx", line 313, in pyfmi.fmi.ModelBase._exec_simulate_algorithm
  File "/opt/conda/lib/python3.8/site-packages/pyfmi/fmi_algorithm_drivers.py", line 385, in __init__
    self.probl = FMIODE2(self.model, result_file_name=self.result_file_name, with_jacobian=self.with_jacobian, start_time=self.start_time,logging=self.options["logging"], result_handler=self.result_handler,extra_equations=self.options["extra_equations"])
  File "src/pyfmi/simulation/assimulo_interface.pyx", line 592, in pyfmi.simulation.assimulo_interface.FMIODE2.__init__
  File "src/pyfmi/fmi.pyx", line 7705, in pyfmi.fmi.FMUModelME2._get_continuous_states
pyfmi.fmi.FMUException: Failed to retrieve the continuous states.
make: *** [Makefile:11 : run] Erreur 1
```

The exact version of each package is:
```
#21 [16/16] RUN conda list
#21 sha256:0548617c3561e7b24e7586b50e78650f97787acbdc5d1b954ea4844d7efca80e
#21 1.205 # packages in environment at /opt/conda:
#21 1.205 #
#21 1.205 # Name                    Version                   Build  Channel
#21 1.205 _libgcc_mutex             0.1                 conda_forge    conda-forge
#21 1.205 _openmp_mutex             4.5                       1_gnu    conda-forge
#21 1.205 assimulo                  3.2.3            py38hd6244a5_3    conda-forge
#21 1.205 brotlipy                  0.7.0           py38h27cfd23_1003  
#21 1.205 ca-certificates           2020.12.5            ha878542_0    conda-forge
#21 1.205 certifi                   2020.12.5        py38h578d9bd_1    conda-forge
#21 1.205 cffi                      1.14.5           py38h261ae71_0  
#21 1.205 chardet                   4.0.0           py38h06a4308_1003  
#21 1.205 conda                     4.9.2            py38h578d9bd_0    conda-forge
#21 1.205 conda-package-handling    1.7.2            py38h03888b9_0  
#21 1.205 cryptography              3.3.1            py38h3c74f83_1  
#21 1.205 fmilib                    2.2.3                he1b5a44_0    conda-forge
#21 1.205 icu                       68.1                 h58526e2_0    conda-forge
#21 1.205 idna                      2.10               pyhd3eb1b0_0  
#21 1.205 ld_impl_linux-64          2.33.1               h53a641e_7  
#21 1.205 libblas                   3.9.0                8_openblas    conda-forge
#21 1.205 libcblas                  3.9.0                8_openblas    conda-forge
#21 1.205 libedit                   3.1.20191231         h14c3975_1  
#21 1.205 libffi                    3.3                  he6710b0_2  
#21 1.205 libgcc-ng                 9.3.0               h2828fa1_18    conda-forge
#21 1.205 libgfortran-ng            7.5.0               h14aa051_18    conda-forge
#21 1.205 libgfortran4              7.5.0               h14aa051_18    conda-forge
#21 1.205 libgomp                   9.3.0               h2828fa1_18    conda-forge
#21 1.205 libiconv                  1.16                 h516909a_0    conda-forge
#21 1.205 liblapack                 3.9.0                8_openblas    conda-forge
#21 1.205 libopenblas               0.3.12          pthreads_hb3c22a3_1    conda-forge
#21 1.205 libstdcxx-ng              9.1.0                hdf63c60_0  
#21 1.205 libxml2                   2.9.10               h72842e0_3    conda-forge
#21 1.205 libxslt                   1.1.33               h15afd5d_2    conda-forge
#21 1.205 lxml                      4.6.2            py38hf1fe3a4_1    conda-forge
#21 1.205 metis                     5.1.0             h58526e2_1006    conda-forge
#21 1.205 ncurses                   6.2                  he6710b0_1  
#21 1.205 numpy                     1.20.1           py38h18fd61f_0    conda-forge
#21 1.205 openssl                   1.1.1j               h7f98852_0    conda-forge
#21 1.205 pip                       21.0.1           py38h06a4308_0  
#21 1.205 pycosat                   0.6.3            py38h7b6447c_1  
#21 1.205 pycparser                 2.20                       py_2  
#21 1.205 pyfmi                     2.8.5            py38h5c078b8_0    conda-forge
#21 1.205 pyopenssl                 20.0.1             pyhd3eb1b0_1  
#21 1.205 pysocks                   1.7.1            py38h06a4308_0  
#21 1.205 python                    3.8.5                h7579374_1  
#21 1.205 python_abi                3.8                      1_cp38    conda-forge
#21 1.205 readline                  8.1                  h27cfd23_0  
#21 1.205 requests                  2.25.1             pyhd3eb1b0_0  
#21 1.205 ruamel_yaml               0.15.87          py38h7b6447c_1  
#21 1.205 scipy                     1.5.3            py38h828c644_0    conda-forge
#21 1.205 setuptools                52.0.0           py38h06a4308_0  
#21 1.205 six                       1.15.0           py38h06a4308_0  
#21 1.205 sqlite                    3.33.0               h62c20be_0  
#21 1.205 suitesparse               5.7.2                h717dc36_0    conda-forge
#21 1.205 sundials                  5.4.0                h6d3a9b8_1    conda-forge
#21 1.205 tbb                       2020.2               hc9558a2_0    conda-forge
#21 1.205 tk                        8.6.10               hbc83047_0  
#21 1.205 tqdm                      4.56.0             pyhd3eb1b0_0  
#21 1.205 urllib3                   1.26.3             pyhd3eb1b0_0  
#21 1.205 wheel                     0.36.2             pyhd3eb1b0_0  
#21 1.205 xz                        5.2.5                h7b6447c_0  
#21 1.205 yaml                      0.2.5                h7b6447c_0  
#21 1.205 zlib                      1.2.11               h7b6447c_3  
#21 DONE 1.3s
```

--- a/src/pyfmi/fmi.pyx
+++ b/src/pyfmi/fmi.pyx
@@ -7921,7 +7921,7 @@ cdef class FMUModelME2(FMUModelBase2):
         if self._nContinuousStates > 0:
             return FMIL.fmi2_import_get_continuous_states(self._fmu, &ndx[0] ,self._nContinuousStates)
         else:
-            return FMIL.fmi2_import_get_continuous_states(self._fmu, NULL ,self._nContinuousStates)
+            return FMIL.fmi2_status_ok
 
     def _get_continuous_states(self):
         """

